llama_stack/distribution/build_container.sh

```diff
@@ -23,7 +23,6 @@ special_pip_deps="$6"
 set -euo pipefail
 
 build_name="$1"
-image_name="distribution-$build_name"
```

Comment:
fixes the distribution-distribution docker images

Reply:
I am not sure if this is the root cause though since this line existed before and my PR didn't change this.

Reply:
root cause is there's 2 places with the 'distribution-' prefix:
1. https://github.com/meta-llama/llama-stack/blob/55067fa81df78ed7a28aa77e9b8d0676cd987f81/llama_stack/distribution/build_container.sh#L25-L26

2. https://github.com/meta-llama/llama-stack/blob/55067fa81df78ed7a28aa77e9b8d0676cd987f81/llama_stack/cli/stack/_build.py#L241-L266C40

(2) passes image_name as argument to (1) so we don't need (1). 

This issue is introduced in https://github.com/meta-llama/llama-stack/pull/788 as we previously don't pass (2) to (1).

Reply:
Sounds good!